test.sh

```diff
@@ -393,15 +393,41 @@ function test-build-and-run-client-server-perf-test()
         num_msgs_per_client=$1
     fi
 
+    local l3_log_disabled=0
+    local l3_log_enabled=1
+    local l3_LOC_disabled=0
+
     echo " "
     echo "${Me}: Client-server performance testing with L3-logging OFF:"
     echo " "
-    build-and-run-client-server-perf-test "${num_msgs_per_client}" 0
+    build-and-run-client-server-perf-test "${num_msgs_per_client}"      \
+                                          "${l3_log_disabled}"
 
     echo " "
     echo "${Me}: Client-server performance testing with L3-logging ON:"
     echo " "
-    build-and-run-client-server-perf-test "${num_msgs_per_client}" 1
+    build-and-run-client-server-perf-test "${num_msgs_per_client}" \
+                                          "${l3_log_enabled}"
+
+    local nentries=100
+    echo " "
+    echo "${Me}: Run L3-dump script to unpack log-entries. (Last ${nentries} entries.)"
+    echo " "
+
+    set -x
+    # No LOC-encoding is in-effect, so no need for the --loc-binary argument.
+    ./l3_dump.py                                                                \
+            --log-file /tmp/l3.c-server-test.dat                                \
+            --binary "./build/${Build_mode}/bin/use-cases/svmsg_file_server"    \
+        | tail -${nentries}
+
+    echo " "
+    echo "${Me}: Client-server performance testing with L3-fast-logging ON:"
+    echo " "
+    build-and-run-client-server-perf-test "${num_msgs_per_client}"  \
+                                          "${l3_log_enabled}"       \
+                                          "${l3_LOC_disabled}"      \
+                                          "fast"
```

Comment:
We want to run the exercise with L3-logging enabled, without tracking extra LOC-ID, -but- we do want fast logging ON.